src/message.ts

```diff
@@ -441,20 +441,21 @@ export class Message implements Sayable {
       return contactList
     }
 
-    // define magic code `8197` to identify @xxx
-    const AT_SEPRATOR = String.fromCharCode(8197)
+    // define magic code `8197` to identify @xxx now it is 32(normal blank)
+    const AT_SEPRATOR = String.fromCharCode(32)
 
     const atList = this.content().split(AT_SEPRATOR)
 
     if (atList.length === 0) return contactList
 
     // Using `filter(e => e.indexOf('@') > -1)` to filter the string without `@`
-    const rawMentionedList = atList
+    let mentionList: string[] = []
+    atList
```

Comment:
`atList` served for function `multipleAt(str)`, function `multipleAt(str)` to conver atList str to mentionList. I add commenti in the code.